python/lsst/jointcal/utils.py

```diff
@@ -183,7 +188,7 @@ def rms_per_source(data):
                            self.old_dist_total.relative, self.old_dist_total.absolute,
                            self.new_dist_total.relative, self.new_dist_total.absolute, name, outdir=outdir)
 
-        plot_all_wcs_deltas(plt, data_refs, visit_catalogs, old_wcs_list, name,
+        plot_all_wcs_deltas(plt, data_refs, self.visits_per_dataRef, old_wcs_list, name,
                             outdir=outdir, per_ccd_plot=per_ccd_plot)
 
         if interactive:
```

Comment:
Yes: the goal is for interactive mucking with the plots and poking at the data, which only happens if the interactive flag is set (defaults to false).